CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 #### Changes
+* Java: Add test for GlideString ([#2271](https://github.com/valkey-io/valkey-glide/pull/2271))
```

Comment:
Move this to "Fixes".  
Update CHANGELOG description to describe the change. 
Change the PR description to reflect what you're fixing.  e.g. Fixing data conversion from bytes